src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  let total = 0;
```

Comment:
I think less variables is good thing in terms of space complexity, may be we can avoid variable total by directly return statements like
return days * costPerDay;

Reply:
done, good suggestion..